.gitignore

```diff
@@ -17,9 +17,6 @@
 .pnp.*
 
 # Artifacts from running the daemons
-/maker.sqlite
-/maker_wallet_db/
-/taker.sqlite
-/taker_wallet_db/
+/*.sqlite*
```

Comment:
nice - it will also take care of the temporariy files that I forgot to include earlier 👍